### ISSUE TITLE
[FEAT] Add (i) About - description/disclaimer

### ISSUE THREAD

**Issue by charlottegiseleweil** [EPFL-ENAC/urbes-viz#1]
Background/ackhnologments / Status (under developmetn) -> link to report bugs + data sources etc

**Comment by GuoshiuanLin** [comment]
More details: Add data source (listed below) under 'info' tab, add color bar, and change layer names to those in parantheses

Buildings layer (Buildings): Swiss Federal Office of Topography

Weather data (Urban climate: temperature): credit: Aldo Brandi, URBES. Perhaps change the colormap to blue-red (cold-warm)

Add another layer with the wind speed from the  with the layer name: Urban climate (wind). Perhaps these 2 layers can be under the same 'Urban climate' layer with 2 little tabs temperature and wind. credit: Aldo Brandi, Urbes. The colormap uses the current one

Areas layer (Areas): What is this layer. I don't see it?

Roads layer (Roads): Swiss Federal Office of Topography

Swiss roads statistics (Traffic 2017): Swiss Federal Office for Spatial Development

Projected Swiss roads statistics (Traffic 2050): Swiss Federal Office for Spatial Development

GWS data (Building numbers): Swiss Federal Statistical Office

Statistical population data (Population): Swiss Federal Statistical Office

Data sources can be appeared when users click a small ‘info’ button

For each layer, add a minimal color bar.

**Comment by pierreleripoll** [comment]
@GuoshiuanLin @ManoliG I updated the app during holidays, could verify it's ok for you ?

**Comment by ManoliG** [comment]
That's great, thanks @pierreleripoll ! Just a couple of final changes needed: (i) there is no legend for wind and temperature and (ii) the animation is fantastic but the marker on the timeline does not move. Everything else looks perfect!

**Comment by pierreleripoll** [comment]
Happy you like it ! For the (i) for wind and temperature is it necessary to add it for both @ManoliG ? Since it's part of the "Urban Climate" I tought having it for Urban Climate was enough. But I can add it very easily so tell me.

Didn't notice the marker not moving, will fix it asap :)

**Comment by ManoliG** [comment]
@pierreleripoll yes, let's add one legend per variable (i.e., one for temperature and one for wind). Thanks!

**Comment by pierreleripoll** [comment]
@ManoliG oh there was as misunderstanding from my part, but I finally understood what you meant and added the legend for temperature and wind haha
Let me know if it's ok for you !
Also fixed the marker, should be working

**Comment by ManoliG** [comment]
@pierreleripoll Fantastic, thanks! Just double check the colorscale of temperature (not sure it matches the legend - it seems greenish while the temperature colorbar is yellow-purple - or maybe it's ok). But everything else is perfect!

**Comment by pierreleripoll** [comment]
@ManoliG you're too fast haha!! I was fixing it while you took a look I think, so you can check again :)

**Comment by ManoliG** [comment]
@pierreleripoll ahah, sorry! :) The colors of temperature still seem to follow the colorbar of wind - it would be nice to have a different colorpalette (as for the legends).

Btw, this is the exhibition where they will show it: https://www.iuav.it/it/evento?ft=1&id=datapoiesis
You can also add a line somewhere with your name/IT4R if you want - up to you

**Comment by pierreleripoll** [comment]
@ManoliG might be a cache issue, it will eitheir be good in a little time (tomorrow I think) or you can clear your browser cache if you know how to do it !

**Comment by ManoliG** [comment]
Hi @pierreleripoll , I am afraid the colorscale is still not working properly for temperature. Now it's ok only for some timesteps, but over time colors go back and forth between the old and new colorscale

**Comment by pierreleripoll** [comment]
@ManoliG ok I changed the whole path to access WRF data, normally there's should be no cache problem anymore ! Let me know if it persists on your computer !!

**Comment by ManoliG** [comment]
All perfect now! Thanks @pierreleripoll !!

**Comment by pierreleripoll** [comment]
Great! Then I'll close the issue but feel free to open a new issue if needed
